src/RamRelation.h

```diff
@@ -93,6 +100,10 @@ class RamRelationIdentifier {
         return input;
     }
 
+    bool isNullary() const { 
+        return nullary; 
```

Comment:
I thinki it would be nicer if this is taken from the number of attributes, not an extra field.